### DISCUSSION TITLE
Can I delete a remote file during merging?

### DISCUSSION THREAD

**Discussion by pandotcodes** [iterate-ch/cyberduck#16030] [Q&A]
Hello! I am currently downloading several ~500GB files from a Backblaze B2 bucket. The downloads have finished, and Cyberduck has been working on merging the segments for literal days. For cost reasons, I would very much like to delete those files from the bucket now, instead of keeping them there for much longer. Intuitively, since the download is done, I should be able to do that, but often there's some kind of behaviour in software like this that I can't anticipate, so I figured I'd ask about it here.

**Comment by dkocher** [comment]
You may want to disable [^1] segmented downloads if merging the files takes too long on your computer. The file on the server is no longer required when file segments are merged but I would nevertheless be cautious and only delete when you have verified the segments have successfully merged.

[^1]: https://docs.cyberduck.io/cyberduck/download/#segmented-downloads
